pkg/reconciler/kubernetes/tektonhub/tektonhub.go

```diff
@@ -175,12 +175,8 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, th *v1alpha1.TektonHub)
 
 	th.SetDefaults(ctx)
 
-	// reconcile target namespace
-	if err := common.ReconcileTargetNamespace(ctx, nil, th, r.kubeClientSet); err != nil {
-		logger.Errorw("error on reconciling targetNamespace",
-			"targetNamespace", th.Spec.GetTargetNamespace(),
-			err,
-		)
+	tc, _ := r.operatorClientSet.OperatorV1alpha1().TektonConfigs().Get(ctx, v1alpha1.ConfigResourceName, metav1.GetOptions{})
```

Comment:
I still feel this would not work when TektonConfig is nil

Reply:
can you please explain the case when TektonConfig could be nil, so I can check once this ?

Reply:
Meaning TektonHub is independent of TekttonConfig, so ideally we should not check for the namespace of TektonConfig, we can directly check if the namespace mentioned in TektonHub is present or not and based on that we can directly create that

For ref: https://github.com/tektoncd/operator/commit/51fff501ad3175a5a75721625d7f5384209b1b77#diff-1c3da4772ce1a38a45935ea0814741d91d7803c96d9aef2b0f10b00f6d8b33f3L864-L875

Reply:
okay, thank you